src/Adapters/Auth/AuthAdapter.js

```diff
@@ -1,20 +1,102 @@
 /*eslint no-unused-vars: "off"*/
+
+/**
+ * @interface ParseAuthResponse
+ * @property {Boolean} [doNotSave] If true, Parse Server will do not save provided authData.
+ * @property {Object} [response] If set, parse server will send the provided response to the client under authDataResponse
+ * @property {Object} [save] If set, parse server will save the object provided into this key, instead of client provided authData
```

Comment:
```suggestion
 * @property {Object} [save] If set, Parse Server will save the object provided into this key, instead of client provided authData
```